### PR TITLE
Update docs build instruction

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Update Docs instructions.
resolve https://github.com/ynput/ayon-documentation/issues/234

This PR was inspired by https://github.com/ynput/ayon-documentation/pull/223

## Testing notes:
follow the building instructions in the PR and confirm it works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,24 @@ If you spot something incorrect or just think a page could do with some improvem
...
 ## Build
+### Configure Environment
+To get started, ensure you have the following:
 
-- Install Node.js https://nodejs.org/en/download
-- `npm install -g yarn`
-- `yarn add docusaurus --dev`
-- `cd {ayon-doc-repo}/website`
-- `yarn start`
+- Download and install Node.js from [nodejs.org](https://nodejs.org/en/download).
```
Comment (BigRoy) [inline README.md:24]:
```suggestion
- Download and install [Node.js](https://nodejs.org/en/download).
```
Would this suffice too?

**Review by BigRoy — COMMENTED** [review]
For what it's worth - I preferred the old instructions. To the point, and simple command line stuff I could almost directly copy paste into my terminal. Now I need to search through a lot of text to see what I should do and copy each line individually.

I'd rather keep the old command line instruction - and maybe ONLY add this below it as 'explanation'.

**Review by Innders — COMMENTED** [review]
I understand why you would want to explain each step but I think Roy is right.

Also these are developers that are using this repo, they should have a rough idea of what each command does and if not then they can look it up.

**Comment by MustafaJafar** [comment]
Thank you for your feedback.
Closing this PR.

**Pull request closed without merging**